### DISCUSSION TITLE
Key names for dpad directional

### DISCUSSION THREAD

**Discussion by ebbisis** [sezanzeb/input-remapper#726] [General]
Hello,

Im having trouble finding the for the key names for the dpad on a controller. The up , down , left , right key names. I want to map one of the paddles on the back of the controller to a dpad direction.  
I tried input-remapper-control --symbol-names , but im not sure what im looking for there. I couldn't see it suggested in the gtk application either.
I had a look at thread #464 it indicates the analog event for what I am assuming is the thumb stick , but ideally I would like to map the none analog dpad input.
Any help appreciated.

Kind Regards,
ebbii

**Comment by jonasBoss** [comment]
As I wrote in https://github.com/sezanzeb/input-remapper/discussions/464#discussioncomment-3530908 the d-pad is an analog input, not a button input. So you need to use the solution I described there.

**Reply by ebbisis** [reply]
Got it , thanks. I made assumptions that were incorrect.
